dist/variables/ds4/color-variables.less

```diff
@@ -91,48 +91,47 @@
 @color-filter-selected-background: lighten(@color-b4, 55%);
 @color-filter-selected-hover-background: lighten(@color-b4, 45%);
 
-// dark mode variables are simply dummy variables (so that build doesn't break)
+// ALL DARK MODE VALUES ARE TAKEN FROM DS6
```

Comment:
Should we have `dark-mode` be in it's own separate variable file?

Reply:
Yep. Good idea, as this file is getting quite long. I will do that.

Reply:
So I managed to reshuffle the vast majority of dark mode colour aliases into a shared base file.